src/middleware/errors.json

```diff
@@ -118,22 +108,34 @@
     "userMessages": "",
     "statusCode": 400
   },
+
+
   {
-    "name": "optionsValidationError",
-    "code": "17",
-    "developerMessage": "One of the supplied options is invalid",
-    "statusCode": 400
+    "name": "SequelizeConnectionError",
+    "code": "94",
+    "developerMessage": "Could not make connection to Sequelize, check authentication rules on database and credentials provided to app",
+    "userMessages": "",
+    "statusCode": 500
   },
   {
-    "name": "updateRegistrationNotFoundError",
-    "code": "18",
-    "developerMessage": "The registration application reference specified could not be found for the council requested. Please check this reference is definitely associated with this council",
-    "statusCode": 404
+    "name": "notifyMissingKey",
+    "code": "95",
```

Comment:
Is the jump in code numbers just to help distinguish between reg and collections service?

Reply:
These ones are notify errors, which should only be seen internally by our notification tasks.
They did have conflicting codes with the collections service, so moved them up to avoid conflicts and distinguish as errors that wouldn't be seen externally.